Mathlib/Combinatorics/Quiver/ConnectedComponent.lean

```diff
@@ -24,123 +22,7 @@ universe v u
-    reverse (reverse f) = f := by apply h.inv'
-
-variable {V}
-
-instance : HasReverse (Symmetrify V) :=
-  ⟨fun e ↦ e.swap⟩
-
-instance :
-    HasInvolutiveReverse
-      (Symmetrify V) where
-  toHasReverse := ⟨fun e ↦ e.swap⟩
-  inv' e := congr_fun Sum.swap_swap_eq e
-
-/-- Reverse the direction of a path. -/
-@[simp]
-def Path.reverse [HasReverse V] {a : V} : ∀ {b}, Path a b → Path b a
-  | _, Path.nil => Path.nil
-  | _, Path.cons p e => (Quiver.reverse e).toPath.comp p.reverse
-
-@[simp]
-theorem Path.reverse_toPath [HasReverse V] {a b : V} (f : a ⟶ b) :
-    f.toPath.reverse = (Quiver.reverse f).toPath :=
-  rfl
-#align quiver.path.reverse_to_path Quiver.Path.reverse_toPath
-
-@[simp]
-theorem Path.reverse_comp [HasReverse V] {a b c : V} (p : Path a b) (q : Path b c) :
-    (p.comp q).reverse = q.reverse.comp p.reverse := by
-  induction' q with _ _ _ _ h
-  · simp
-  · simp [h]
-
-@[simp]
-theorem Path.reverse_reverse [h : HasInvolutiveReverse V] {a b : V} (p : Path a b) :
-    p.reverse.reverse = p := by
-  induction' p with _ _ _ _ h
-  · simp
-  · rw [Path.reverse, Path.reverse_comp, h, Path.reverse_toPath, Quiver.reverse_reverse]
-    rfl
-
-/-- The inclusion of a quiver in its symmetrification -/
-def Symmetrify.of : Prefunctor V (Symmetrify V) where
-  obj := id
-  map := Sum.inl
-
-/-- Given a quiver `V'` with reversible arrows, a prefunctor to `V'` can be lifted to one from
-    `Symmetrify V` to `V'` -/
-def Symmetrify.lift {V' : Type _} [Quiver V'] [HasReverse V'] (φ : Prefunctor V V') :
-    Prefunctor (Symmetrify V) V' where
-  obj := φ.obj
-  map f := match f with
-  | Sum.inl g => φ.map g
-  | Sum.inr g => reverse (φ.map g)
-
-theorem Symmetrify.lift_spec (V' : Type _) [Quiver V'] [HasReverse V'] (φ : Prefunctor V V') :
-    Symmetrify.of.comp (Symmetrify.lift φ) = φ := by
-  fapply Prefunctor.ext
-  · rintro X
-    rfl
-  · rintro X Y f
-    rfl
-
-theorem Symmetrify.lift_reverse (V' : Type _) [Quiver V'] [h : HasInvolutiveReverse V']
-    (φ : Prefunctor V V') {X Y : Symmetrify V} (f : X ⟶ Y) :
-    (Symmetrify.lift φ).map (Quiver.reverse f) = Quiver.reverse ((Symmetrify.lift φ).map f) := by
-  dsimp [Symmetrify.lift]; cases f
-  · simp only
-    rfl
-  · simp only
-    rw [h.inv']
-    rfl
-
-/-- `lift φ` is the only prefunctor extending `φ` and preserving reverses. -/
-theorem Symmetrify.lift_unique (V' : Type _) [Quiver V'] [HasReverse V'] (φ : Prefunctor V V')
-    (Φ : Prefunctor (Symmetrify V) V') (hΦ : Symmetrify.of.comp Φ = φ)
-    (hΦinv : ∀ {X Y : Symmetrify V} (f : X ⟶ Y),
-      Φ.map (Quiver.reverse f) = Quiver.reverse (Φ.map f)) :
-    Φ = Symmetrify.lift φ := by
-  subst_vars
-  fapply Prefunctor.ext
-  · rintro X
-    rfl
-  · rintro X Y f
-    cases f
-    · rfl
-    · exact hΦinv (Sum.inl _)
-#align quiver.symmetrify.lift_unique Quiver.Symmetrify.lift_unique
-
-variable (V)
+variable (V : Type _) [Quiver.{u+1} V]
```

Comment:
```suggestion
variable (V : Type u) [Quiver.{v+1} V]
```
to match the mathlib PR. Feel free to do the opposite change to the mathlib PR if you prefer.

Reply:
OK, I'll change the mathlib one, because I feel like the fewer universe choices we have, the fewer opportunities to make mistakes we'll have to deal with.